Code/DataStructs/RealValueVect.cpp

```diff
@@ -91,78 +88,77 @@ void RealValueVect::initFromText(const char *pkl, const unsigned int len) {
   ss.write(pkl, len);
   boost::int32_t tVers;
   streamRead(ss, tVers);
-  tVers *= -1;
-  if (tVers == 0x1) {
-  } else {
+  tVers = -tVers;
```

Comment:
why?

Reply:
That's part of my campaign to check if your reactions to code changes are reproducible. I always do that change and you always complain. I personally find the `a = -a` version clearer. It makes no difference otherwise, as demonstrated here a long time ago:
https://stackoverflow.com/questions/15111466/fastest-way-to-negate-a-number
But again, it is mostly an alertness and reproducibility check, which you have brilliantly passed. So I am happy to revert it if you prefer the `*= -1` version :-)

Reply:
Just reverted it.